execution/evm/vm.go

```diff
@@ -210,6 +210,8 @@ func (vm *VM) Call(caller, callee acm.MutableAccount, code, input []byte, value
 				return nil, fmt.Errorf("error transferring value %v %s (callee) -> %s (caller)",
 					value, callee, caller)
 			}
+		} else {
+			childCallState.Sync(callState)
```

Comment:
I'm starting to feel `Sync` was a poor choice of name - that said I'm not sure i can think of a better one. Perhaps a comment here would be a nice addition:
```go
// Copy any state updates from child call frame into current call frame
```